src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -415,9 +416,21 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<MarkDownBlock>()
                 .Property(block => block.Body)
                 .HasColumnName("Body");
+            
+            modelBuilder.Entity<Permalink>()
+                .Property(permalink => permalink.Created)
+                .HasConversion(
+                    v => v,
+                    v => DateTime.SpecifyKind(v, DateTimeKind.Utc));
+
+            modelBuilder.Entity<Permalink>()
+                .HasIndex(data => data.ReleaseId);
+
+            modelBuilder.Entity<Permalink>()
+                .HasIndex(data => data.SubjectId);
 
             modelBuilder.Entity<ReleaseContentSection>()
-                .HasKey(item => new {item.ReleaseId, item.ContentSectionId});
+            .HasKey(item => new {item.ReleaseId, item.ContentSectionId});
```

Comment:
A bit o' formatting gone awry here

Reply:
Thanks, I've edited the original commit to fix this.